src/components/Books.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const Books = () => (
```

Comment:
- [ ] Kindly create a new component (eg `Book`) that represents a single book. This component should take in book data as a prop and render its title and author
- [ ] Kindly ensure that the new aforementioned component (eg `Book`) has a placeholder Remove button (this button does not have to work yet - it is merely a placeholder)
- [ ] Your **`Books`** component (which you already have) should render a list of the created `Book` components

Reply:
The second and the third points have been done well. However, one request is not yet met. Kindly check below for more information.

Reply:
> The second and the third points have been done well. However, one request is not yet met. Kindly check below for more information.

@peterrobert Hello

Reply:
All changes have been done well